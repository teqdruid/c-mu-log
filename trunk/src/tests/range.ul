/* test to find how the range cases work*/

foo($x,$y) {
    10 >= $x;
    1 <= $x;
     $y>9;
    19 >$y;   
}

bar($z) {
    $z < 50;
    10 < $z;
}

main() {
    {OR:
	foo($x,$y);
	bar($z);	
	@print("the solutions for $x ",$x,
	       " the solutions for $y ",$y,
	       " the solutions for $z ",$z);
    }
    
		
}
/* conclusions: If an infinte range is given the interpretor gives no solution
		The values returned are exclusive in a range
		so for 	10 >= $x;
    			1 <= $x;	
		the interpreter returns a range 0..11
*/

