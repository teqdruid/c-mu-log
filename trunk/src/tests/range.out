Solution
Solution
Solution
Solution
Solution
Solution
No more solutions
